services-js/access-boston/src/client/device-verification/DeviceVerificationModal.tsx

```diff
@@ -207,7 +214,16 @@ export default class DeviceVerificationModal extends React.Component<Props> {
             try a different number or email
           </button>.
         </div>
-      </>
+      </form>
     );
   };
 }
+
+function elidePhoneNumber(phoneNumber: string | null): string {
```

Comment:
I feel like this could be a useful function to keep _some_ where for future use... (gitbook? react-fleet? an as-of-yet nonexistent `cob-utility-functions` module?)

Reply:
yep, we should factor it out next time we need it.